src/components/Select/index.tsx

```diff
@@ -10,15 +10,17 @@ export type SelectOption = {
   onClick?: () => void
 }
 
+export type ReactSelectElement = ReactSelect<SelectOption>
```

Comment:
we define a less-precise version of this in the console; it will be helpful to get it directly from the library